modules/builtin/src/content-types/_utils.js

```diff
@@ -17,7 +17,7 @@ function isUrl(str) {
 }
 
 function formatURL(baseUrl, url) {
-  if (isBpUrl(url)) {
+  if (!url.startsWith('http')) {
```

Comment:
If it doesn't start with /api, then add the base URL. So if you provide the URL `{{temp.link}}` it will return http://localhost:3000/{{temp.link}}

Reply:
if it does start with `/api` we also want to add the baseURL. Could you elaborate on the conditional to add here?